JeongJaeHeon/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "jeongjaeheon",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql": "^2.18.1",
+    "mysql2": "^2.3.3",
+    "nodemon": "^2.0.20",
+    "typeorm": "^0.3.10"
+  }
```

- express, mysql, typeorm, dotenv, cors와 같은 패키지는 사용자에게 서비스되는 코드에서는 반드시 포함되어야 하는 라이브러리이므로 package.json 내에 `dependencies`에 포함되어야 합니다.
- 하지만, nodemon과 같은 패키지는 로컬에서 개발할 때, 생산성을 위해서 사용하는 패키지입니다. 그렇기 때문에 이는 `devDependencies`에 추가 혹은 `global 환경`에 설치해서 사용할 수 있도록 만들어주세요.
- mysql & mysql2 드라이버 패키지는 서로 동일한 기능을 하는 패키지 입니다. 따라서 둘 중 하나를 선택하여 설치를 진행해주세요.

패키지 설치/삭제와 관련된 명령어를 참고하여 수정해주세요!
- npm install --save "package-name"
- npm install --save-dev "packagae-name"
- npm uninstall "package-name"
 
```suggestion
  "dependencies": {
    "cors": "^2.8.5",
    "dotenv": "^16.0.3",
    "express": "^4.18.2",
    "mysql": "^2.18.1",
    "typeorm": "^0.3.10"
  }.
  devDependencies: {
    "nodemon": "^2.0.20",
    "morgan": "^1.10.0"
  }
```